openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -145,6 +145,34 @@
   height: 100%;
 }
 
+// Padding
+.p-0 {
+  padding: 0px;
+}
+.p-1 {
+  padding: 4px;
+}
+.p-4 {
+  padding: 16px;
+}
+.pt-4 {
+  padding-top: 16px;
+}
+
+// Margin
+.m-0 {
+  margin: 0px;
+}
+.m-1 {
+  margin: 4px;
+}
+.m-4 {
+  margin: 16px;
+}
+.mt-4 {
+  margin-top: 16px;
+}
+
```

Comment:
move this to space.less

Reply:
Removed these and used already defined class in spacing.less